docs/book/src/user/quick-start.md

```diff
@@ -621,9 +621,11 @@ clusterctl generate cluster capi-quickstart --flavor development \
 {{#/tab }}
 {{#/tabs }}
 
-This creates a YAML file named `capi-quickstart.yaml` with a predefined list of Cluster API objects; Cluster, Machines,
+This creates a YAML file named [capi-quickstart.yaml] with a predefined list of Cluster API objects; Cluster, Machines,
 Machine Deployments, etc.
 
+The sample file can be seen here 
```

Comment:
Can you link the file on this line or else make the link in the previous line more explicit?